template/analytics.notify.version/body.html

```diff
@@ -99,18 +99,14 @@
           <th style="text-align: center;">Current Version</th>
           <th style="text-align: center;">Latest Version</th>
         </tr>
+        {{ range $depsInd, $dependency := .custom.version_updates }}
         <tr>
-          <td>#01</td>
-          <td>com.logback.core</td>
-          <td style="text-align: center;">2.0</td>
-          <td style="text-align: center;">2.1</td>
-        </tr>
-        <tr>
-          <td>#02</td>
-          <td>com.logback.web</td>
-          <td style="text-align: center;">1.5</td>
-          <td style="text-align: center;">1.8</td>
+          <td>#{{inc $depsInd}}</td>
+          <td>{{$dependency.name}}</td>
+          <td style="text-align: center;">{{$dependency.version}}</td>
+          <td style="text-align: center;">{{$dependency.latest_version}}</td>
         </tr>
+        {{ end }}
```

Comment:
Check line-80 [link](https://github.com/fabric8-services/fabric8-notification/pull/84/files#diff-be106f92fb7465db3762f25f5e4da3abL80), it seems that you missed to use `.custom.repo_url`